### PR TITLE
23080:"Modified: " date stamp on comments.

### PR DESCRIPTION
**Pull request by don-aot**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml [hunk 1]
```diff
@@ -48,7 +48,15 @@
                             @comment.Comment
                         </div>
                     </div>
-                    <div class="commented-time mt-3">@comment.CreationTime.ToString("yyyy-MM-dd h:mm tt")</div>
+                    @if (@comment.LastModificationTime != null)
+                    {
+                        <div class="commented-time mt-3">Modified: @comment.LastModificationTime?.ToString("yyyy-MM-dd h:mm tt")</div>
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml:53]: lets change the modified to a language key: (L:modified) or something to that effect

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml [hunk 2]
```diff
@@ -48,7 +48,15 @@
...
+                    }
+                    else
+                    {
+                        <div class="commented-time mt-3">@comment.CreationTime.ToString("yyyy-MM-dd h:mm tt")</div>
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml:57]: Should we not add the (L:created) in front of this timestamp

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml (hunk 1 above)
Comment (don-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml]: @AndreGAot  Updated

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml (hunk 2 above)
Comment (don-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/Views/Shared/Components/CommentsWidget/Default.cshtml]: @AndreGAot  Updated

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**